flavors/lineageos/update.sh

```diff
@@ -21,7 +21,7 @@ fi
 args=(
     --ref-type branch
     "https://github.com/LineageOS/android"
-    "lineage-17.1"
+    "lineage-18.1"
```

Comment:
Since we would want to allow the user to update each branch independently, we should take the branch as an argument to the script.